cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -672,22 +672,31 @@ func main() {
 		debug = log.New(os.Stderr, "", log.LstdFlags)
 	}
 
-	client := retryablehttp.NewClient()
-	client.Logger = debug
+	var sg Sourcegraph
+	if rootURL.IsAbs() {
```

Comment:
maybe `url.Parse(rootURL).Scheme != ""`? That way it works with relative directories too.